docs/mddocs/Quickstart/npu_quickstart.md

```diff
@@ -0,0 +1,203 @@
+# Install IPEX-LLM on Windows with Intel NPU
+
+IPEX-LLM provides NPU support for LLM acceleration on Intel Core™ Ultra Processers (Series 2), and offers both Python and C++ API. This guide not only demonstrates how to install IPEX-LLM on Windows with Intel NPU, but also includes a quick example of running an LLM on Intel NPU with IPEX-LLM Python/C++ API.
```

Comment:
This guide demonstrates:
- How to install IPEX-LLM for Intel NPU on Intel Core™ Ultra Processers (Series 2)
- Python and C++ APIs for running IPEX-LLM on Intel NPU